nni/retiarii/nn/pytorch/mutator.py

```diff
@@ -116,12 +120,94 @@ def process_inline_mutation(model: Model) -> Optional[List[Mutator]]:
         mutator = LayerChoiceMutator(node_list)
         applied_mutators.append(mutator)
 
-
     if applied_mutators:
         return applied_mutators
     return None
 
 
+# The following are written for pure-python mode
+
+
+class ManyChooseManyMutator(Mutator):
```

Comment:
please add docstring for it

Reply:
This is an internal API. Added a short description.